profiles/common/themes/ec_resp_17/css/text_size_small.css

```diff
@@ -0,0 +1,4 @@
+/**
```

Comment:
"Can we remove this? Maybe not risk it in scope of this issue, there might be modules in subsites including it."